test/specs/rules/disallow-empty-blocks.js

```diff
@@ -125,4 +125,37 @@ describe('rules/disallow-empty-blocks', function() {
     it('should not report empty block for function expressions', function() {
         assert(checker.checkString('var a = function(){};').isEmpty());
     });
+
+    describe('options as object', function() {
+        describe('allExcept as option', function() {
```

Comment:
It seems you could unite those two describes, also, could you call it `allExcept: ["comments"]`, we do that in couple places, i guess that would be good trend to follow.